modules/statistics/php/stats_demographic.class.inc

```diff
@@ -374,60 +386,63 @@ class Stats_Demographic extends \NDB_Form
         );
         $instrument_list = \Utility::getAllInstruments();
         $this->tpl_data['all_instruments'] = $instrument_list;
-        if ($_REQUEST['DemographicInstrument'] ?? ''
-        ) {
+        $demographicInstrument = $_REQUEST['DemographicInstrument'] ?? '';
+        $demographicInstrument = htmlspecialchars($demographicInstrument);
```

Comment:
This can go below the empty check

Reply:
same question as above. its a !empty check